lib/galaxy/datatypes/sequence.py

```diff
@@ -714,6 +714,9 @@ def sniff_prefix(self, file_prefix):
         True
         >>> FastqCSSanger().sniff(fname)
         True
+        >>> fname = get_test_fname('2.fastqsanger')
```

Comment:
Can you avoid filenames that were used for files moved to https://github.com/galaxyproject/galaxy-test-data/ ?

```suggestion
        >>> fname = get_test_fname('4.fastqsanger')
```